### PR TITLE
First commit - initial set up

### PR DESCRIPTION
**Pull request by priscilagonzalez**
It includes:
- Directories created.
- Dependencies installed.
- Server created.
- Basic HTML page and JSX file created.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "url": "git+https://github.com/hrr43fec-luke/video-carousel-service.git"
+  },
+  "author": "Priscila Gonzalez",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hrr43fec-luke/video-carousel-service/issues"
+  },
+  "homepage": "https://github.com/hrr43fec-luke/video-carousel-service#readme",
+  "dependencies": {
+    "@babel/preset-react": "^7.8.3",
```
Comment (Kraig5) [inline package.json]: Minor: this should be a dev dependency I think

**Comment by Kraig5** [comment]
Hi, everything looks good but I suggest adding your bundle to your gitignore. I don't think we're supposed to store generated files in git -- there's no point in reviewing them, etc.

Also, minor, one of your babel dependencies can move down to dev with the others.

File: package.json (hunk 1 above)
Comment (priscilagonzalez) [reply package.json]: Thanks!

Comment (priscilagonzalez) [reply package.json]: Thanks!

**Pull request merged**